python/ribasim/ribasim/schemas.py

```diff
@@ -30,595 +33,755 @@ def migrate(cls, df: Any, schema_version: int) -> Any:
 
 class BasinConcentrationExternalSchema(_BaseSchema):
     fid: Index[Int32] = pa.Field(default=1, check_name=True, coerce=True)
+
```

Comment:
`gen_python.jl` creates even more whitespace, of which ruff now seems to leave one. I tried to fix this with OteraEngine settings but didn't manage. Not so important though.

Reply:
Hmm the codegen CI doesn't generate the whitespace, so I removed it again. Not sure why I get different results.